widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -0,0 +1,1160 @@
+
+        theme.gliaChatBackgroundColor?.let(::getColorCompat)?.also(::setBackgroundColor)
+
+        // fonts
+        theme.fontRes?.also { binding.chatEditText.typeface = getFontCompat(it) }
+
+        // texts
+        theme.appBarTitle?.also(::showToolbar)
+
+        theme.brandPrimaryColor?.also {
+            binding.operatorTypingAnimationView.addColorFilter(color = it)
+        }
+
+        applyTheme(Dependencies.getGliaThemeManager().theme)
+    }
+
+    private fun handleStatusBarColor() {
+        val activity = Utils.getActivity(this.context)
+        if (activity != null && defaultStatusBarColor == null) {
+            defaultStatusBarColor = activity.window.statusBarColor
+            if (controller != null && controller!!.isChatVisible) {
+                activity.window.statusBarColor = statusBarColor
+            }
+        }
+    }
+
+    private fun setupViewActions() {
+        binding.chatEditText.addTextChangedListener(textWatcher)
+        binding.sendButton.setOnClickListener {
+            val message = binding.chatEditText.text.toString().trim { it <= ' ' }
+            controller?.sendMessage(message)
+        }
+        setupAddAttachmentButton()
+        binding.appBarView.setOnBackClickedListener {
+            controller?.onBackArrowClicked(onBackClickedListener)
+        }
+        binding.appBarView.setOnEndChatClickedListener { controller?.leaveChatClicked() }
+        binding.appBarView.setOnXClickedListener { controller?.leaveChatQueueClicked() }
+        binding.newMessagesIndicatorCard.setOnClickListener { controller?.newMessagesIndicatorClicked() }
+    }
+
+    private fun setupAddAttachmentButton() {
+        binding.addAttachmentButton.setOnClickListener {
+            attachmentPopup.show(binding.chatMessageLayout, {
+                val intent = Intent()
+                intent.type = "image/*"
+                intent.action = Intent.ACTION_OPEN_DOCUMENT
+                Utils.getActivity(context).startActivityForResult(
+                    Intent.createChooser(intent, "Select Picture"),
+                    OPEN_DOCUMENT_ACTION_REQUEST
+                )
+            }, {
+                if (ContextCompat.checkSelfPermission(
+                        context,
+                        Manifest.permission.CAMERA
+                    ) == PackageManager.PERMISSION_GRANTED
+                ) {
+                    dispatchImageCapture()
+                } else {
+                    Utils.getActivity(context)
+                        .requestPermissions(
+                            arrayOf(Manifest.permission.CAMERA),
+                            CAMERA_PERMISSION_REQUEST
+                        )
+                }
+            }, {
+                val intent = Intent()
+                intent.type = "*/*"
+                intent.action = Intent.ACTION_OPEN_DOCUMENT
+                Utils.getActivity(context).startActivityForResult(
+                    Intent.createChooser(intent, "Select file"),
```

Comment:
Ideally, better to extract this string to `strings.xml`

Reply:
I made it `constant`, in `strings.xml` it is possible to override it.

Reply:
Fixed - https://github.com/salemove/android-sdk-widgets/pull/396/commits/40a661799a5ee54a4c0cf24786ef79e1216fb79d

Reply:
Why do you want to make it constant and avoid overriding? I imagine that the integrator wants to localize or customize it. Shouldn't they be allowed to do that?

Reply:
I'm sorry I was thinking you meant this `"*/*"`. You're right I will move it to `strings.xml`

Reply:
Fixed - https://github.com/salemove/android-sdk-widgets/pull/396/commits/d9fbb06ba7b097c038990e6782a70b347b6d2fd9